core/src/main/scala/org/ensime/config/ScalariformFormat.scala

```diff
@@ -21,7 +21,12 @@ trait ScalariformFormat {
         descriptor.preferenceType match {
           case BooleanPreference => sexp.convertTo[Boolean]
           case IntegerPreference(_, _) => sexp.convertTo[Int]
-          case IntentPreference => sexp.convertTo[String]
+          case IntentPreference => sexp.convertTo[String].toLowerCase match {
+            case "preserve" => Preserve
+            case "force" => Force
+            case "prevent" => Prevent
+            case intent => sys.error("Unknown scalariform intent: " + intent)
```

Comment:
`error` is a bit harsh for this. There is already an exception type for s-express failures.